spec/models/rbac_spec.rb

```diff
@@ -1,24 +1,10 @@
 require "spec_helper"
 
 describe Rbac do
-  before(:each) do
-    User.stub(:server_timezone => "UTC")
-
-    MiqRegion.seed
-    @tags = {
-      2  => "/managed/environment/prod",
-      3  => "/managed/environment/dev",
-      4  => "/managed/service_level/gold",
-      5  => "/managed/service_level/silver"
-    }
-
-    User.any_instance.stub(:validate => true)
-    @group = FactoryGirl.create(:miq_group)
-    @user  = FactoryGirl.create(:user, :miq_groups => [@group])
-  end
-
   context "tenant scoping" do
     before do
+      MiqRegion.seed
+      User.stub(:server_timezone => "UTC")
       default_tenant = Tenant.seed
```

Comment:
you probably want to call `EvmSpecHelper.create_root_tenant` (that will handle the `MiqRegion.seed` as well)